microsoftgraph/microsoft-graph-explorer-v2
# Updates terms of use text

## Overview

Updates the terms of use info.

### Demo

![image](https://user-images.githubusercontent.com/40403681/67756946-f7e54600-fa4b-11e9-955f-8b0b0a945e2f.png)


### Notes

Closes #262 
Related to: https://github.com/microsoftgraph/microsoft-graph-explorer/issues/359

## Testing Instructions


Notice the above terms of use message (from the screenshot above) placed in between the response and request windows.
Close the message by clicking on the 'close' button at the top right corner.

* Log on to the tst environment.
* Check the Terms of Use info bar.
* Notice the text  **Microsoft Graph API** is changed to  **Microsoft API**
